client/src/App.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import './App.css';
+import MuiThemeProvider from "@material-ui/core/styles/MuiThemeProvider";
+import Button from "@material-ui/core/Button";
+import theme from "./theme";
+import { makeStyles } from '@material-ui/core/styles';
+import Eventcard from './eventcard.js';
+
+
+// function App() {
+//   return (
+//     <div className="App">
+//       <header className="App-body">
+//
+//
+//         <Eventcard />
+//
+//       </header>
+//     </div>
+//   );
+// }
+
+
+class App extends Component {
+state = {
+    data: null
+  };
+
+  componentDidMount() {
+      // Call our fetch function below once the component mounts
+    this.callBackendAPI()
+      .then(res => this.setState({ data: res.express }))
+      .catch(err => console.log(err));
+  }
+    // Fetches our GET route from the Express server. (Note the route we are fetching matches the GET route from server.js
+  callBackendAPI = async () => {
+    const response = await fetch('/express_backend');
+    const body = await response.json();
+
+    if (response.status !== 200) {
+      throw Error(body.message)
+    }
+    return body;
+  };
+
+  render() {
+    return (
+      <div className="App">
+        <header className="App-header">
+          <img src={logo} className="App-logo" alt="logo" />
+          <h1 className="App-title">Welcome to React</h1>
+        </header>
+        // Render the newly fetched data inside of this.state.data
```

Comment:
I typically only leave comments when something cannot be expressed through readable code. For example:

```
const names = ['John', 'Jacob', 'JingleHeimer', 'Smith'];

// Names that don't start with J are considered evil by the J Corporation 
const filteredNames = names.filter(name => name.startsWith('J'));
```

However, I think `<p className="App-intro">{this.state.data}</p>` accurately represents `// Render the newly fetched data inside of this.state.data` on its own.

PS: This comment is totally nitpicky, and I only left it because this is an educational environment. I would not have made it in a work environment, because being nitpicky can alienate your peers.

Reply:
Removed redundant comment. Notes on when to use comments noted.

Reply:
Do you have this change locally and forgot to push it?